subsys/bootloader/include/fw_metadata.h

```diff
@@ -54,6 +70,11 @@ OFFSET_CHECK(struct fw_firmware_info, firmware_version,
 OFFSET_CHECK(struct fw_firmware_info, firmware_address,
 	(CONFIG_SB_MAGIC_LEN + 8));
 
+
+#define ABI_GETTER_OUT_NAME abi_getter_out
+#define ABI_GETTER_IN_NAME  abi_getter_in
```

Comment:
I'd appreciate a comment explaining this.

At first glance the indirection seems unnecessary.

Either we are doing something fancy, or we want to be able to just these names in the future for some reason?

Reply:
This has been removed